.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish snapshot
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - ray/ui-update
+
+jobs:
+  publish-snapshot :
+    runs-on : ubuntu-latest
+    if : github.repository == 'square/workflow-kotlin'
+    timeout-minutes : 25
+
+    steps :
+      - uses: actions/checkout@v2
+      - uses: gradle/wrapper-validation-action@v1
+      - name: set up JDK 11.0.7
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.7
+
+      - uses : burrunan/gradle-cache-action@v1
+        name : Publish Snapshots
+        with :
+          gradle-dependencies-cache-key : |
+            gradle/libs.versions.toml
+          arguments : |
+            clean publish --no-build-cache --no-daemon --stacktrace --no-parallel
```

Comment:
Our release steps use this recipe, w/separate gradle invocations for build and publish. I don't remember what it fixed, maybe no longer an issue. Just a heads up.

```
./gradlew clean build && ./gradlew  publish --no-parallel
```

Reply:
Updated!